src/app/tests/suites/certification/Test_TC_OO_2_2.yaml

```diff
@@ -57,6 +73,14 @@ tests:
       PICS: OO.S.C01.Rsp
       command: "On"
 
+    - label: "Wait 1000ms"
```

Comment:
Do you know this is going to be sufficient for all the DUTs? Is there a limit to how long a device can take to process these commands?

Reply:
There's no a clear definition for that on the TP. my suggestion is that might be turned into a PIXIT variable in future. For now I just leverage the same 1 seconds that some other steps were already waiting and made it standard for all steps.